content/en/docs/user-guides/how-to/timestamping.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Timestamping"
+description: "How to sign and verify artifacts in OCI-compliant registries with timestamping"
+type: docs
+weight: 9
+---
+
+In the X.509 Public Key Infrastructure (PKI) system, digital signatures must be generated within the certificate's validity period, as expired certificates compromise the signature's trustworthiness. The [RFC 3161](https://www.rfc-editor.org/rfc/rfc3161) standard defines the internet X.509 PKI Time-Stamp Protocol (TSP), where a timestamp is issued by a trusted third party acting as a Time Stamping Authority (TSA). These trusted timestamps extend the trust of signatures created within certificates validity, enabling successful signature verification even after certificates have expired.
+
+Since Notation v1.2.0 release, Notation supports RFC 3161 compliant timestamping. During signing, a countersignature is added to the signature for the signing artifact, ensuring the trustworthiness of the timestamp through validation. However, Notation does not use timestamping to verify that the signature was created before the certificate’s revocation, as the revocation time is not deterministic. For example, the revocation time may not coincide with the time the certificates are compromised.

```

Comment:
I think this paragraph is a bit too technical for a guide doc. Since the paragraph at line 8 already describes the background, we can just remove line 10 and start the guide.
```suggestion
```